.circleci/scripts/jvmprocmon-start.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Validated with shellcheck:
```
$ shellcheck ./.circleci/scripts/jvmprocmon-start.sh  && echo "OK"
OK
```

Reply:
Hehe, I just wanted to add a comment like "Did you validate this with shellcheck?" 👍